### ISSUE TITLE
[SUGGESTION] Option to enable PrideGUI on the settings

### ISSUE THREAD

**Issue by Thinkseal** [Pridecraft-Studios/joy#10]

**Comment by blryface** [comment]
It's better if people download from modrinth instead, it makes the mod smaller.

We may have an option for PrideGUI Textures for custom GUIs we potentially make.

**Comment by Ampflower** [comment]
Reopening for consideration.

**Comment by blryface** [comment]
Yeaaah I kinda still think the same thing about this

Pridepack, PrideGUI and Joy should remain separate, although PrideGUI *could* have themed Joy UIs in it

**Comment by Ampflower** [comment]
@NormalFery What would your opinion be for this?

**Comment by NormalFery** [comment]
While it sounds like a *neat* feature, it feels rather useless, or not that good, in my opinion. 

It'd make the mod have 3x the amount of GUIs (Vanilla Joy UI (assuming we end up adding things that need a UI), PrideGUI, and PGUI Light), and if we even wanted to add mod support for PGUI in Joy, that's an even bigger expense with 2x GUIs for every single mod PGUI has support (And there's vanilla UIs too, of course). Along with that, if we just added support for Joy UIs in the mod it'd be a style clash for most people.

I think it'd be better to just add Joy support for Pride GUI on the resource pack instead of adding PGUI to Joy. It'd make things far easier.

**Comment by Ampflower** [comment]
Fair enough. Joy could still house its own GUI variant textures, but, realistically that'd probably be better off in the PrideGUI repo as you say.

I suppose with that, can just not, least for the time being.
